src/accountant.ts

```diff
@@ -79,50 +161,36 @@ export class Accountant {
         return this.client.send(tx.sender.keystore, tx.receiver.address, amount);
     }
 
-    private async processClaim(claim: Claim): Promise<void> {
+    /* eslint-disable  @typescript-eslint/no-explicit-any */
+    private async handleConnectionRetries(f: { (): any },alias: string): Promise<void> {
+      let attempts = 0
+      const maxAttempts = 5
+      for(;;){
         try {
-          await this.client.claim(claim.keystore, this.isDeepHistoryCheckForced, this.gracePeriod);
-        } catch (error: unknown) {
-          this.logger.error(`Could not process the claim for ${claim.alias}: ${error}`);
+          //function
+          await f()
+          return
+        } catch (error) {
+          this.logger.error(`Could not process the claim for ${alias}: ${error}`);
           const message = getErrorMessage(error)
-          if(message.includes('Connection dropped') || message.includes('ECONNRESET')){
+          if(
+            // (
+            //   message.includes('Connection dropped') || 
+            //   message.includes('ECONNRESET') || 
```

Comment:
After a few tries the type of errors were growing, so I decided to embrace them all... there is a maxAttempts number of retries anyway